src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp

```diff
@@ -154,17 +154,21 @@ void CobConverter::UpdateState(uint64_t CurrentSimNanos)
         Eigen::Vector3d rhat_COM_N = dcm_NC * rhat_COM_C;
         Eigen::Vector3d rhat_COM_B = dcm_CB.transpose() * rhat_COM_C;
 
-        /*! - Define diagonal terms of the covariance */
-        Eigen::Matrix3d covar_C;
-        covar_C.setZero();
-        covar_C(0,0) = pow(X,2);
-        covar_C(1,1) = pow(Y,2);
-        covar_C(2,2) = 1;
-        /*! - define and rotate covariance using number of pixels found */
+        /*! - define diagonal terms of the COB covariance */
+        Eigen::Matrix3d covar_cob_C;
```

Comment:
:large_orange_diamond: Variable name doesn't really adhere to code guidelines. Perhaps, `covarCob_C` or similar is better? Happy to discuss of course.

Reply:
Fixed it in the corresponding commit. Also included an additional [Refactor] commit at the end to fix the naming of older variables to adhere to code guidelines.